### PR TITLE
Upgrade embedded icons to v2.0.11

### PR DESCRIPTION
**Pull request by AndrewIngram**
For #141.

There's a bit of final mile work to do, but this should be a reasonable starting point.

A couple of notes:

* Tests that check the rendered SVG data haven't been updated yet, so currently fail. This is primarily because I wasn't sure of the best methodology for updating them, short of copying the output from running the code myself into the test -- which to me isn't really a test.
* Due to the addition of the new mini style, the archive structure is different. I wasn't sure whether the best approach was to remap the new filenames to match the format the code currently expects, or to change the code itself. I opted for the latter for the sake of speed. This means that the template tags for "outline" translates the style name to "24/outline", "solid" to "24/solid" and "mini" to "20/solid" prior to attempting to load from the archive.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = heroicons
-version = 1.8.0
+version = 2.0.0
```
Comment (adamchainz) [inline setup.cfg:3]: My release script will bump this
```suggestion
version = 1.8.0
```

**Review by adamchainz — COMMENTED** [review]
you made `heroicons.zip` by hand? It's generated by `download_heroicons.py`, we'll want to update that script accordingly.

Since we get to select the layout of `heroicons.zip`, I think there's no need for the `24/` and `20/` prefixes, we can flatten those out?

**Comment by adamchainz** [comment]
> Tests that check the rendered SVG data haven't been updated yet, so currently fail. This is primarily because I wasn't sure of the best methodology for updating them, short of copying the output from running the code myself into the test -- which to me isn't really a test.

This is acceptable, it's how I did it. I also checked the output SVG looks okay in a browser.

Hint: you can use [black's preview mode](https://black.readthedocs.io/en/stable/the_black_code_style/future_style.html) to wrap the strings automatically.

**Comment by AndrewIngram** [comment]
> you made `heroicons.zip` by hand? It's generated by `download_heroicons.py`, we'll want to update that script accordingly.

Nah, I ran the script with `python download_heroicons.py 2.0.0`. I'll look at making the script do the zip remapping. Is it important that the script continues to work for the pre-2.0 layout?

**Comment by AndrewIngram** [comment]
I'd suggest a major version bump for this, as the icons sets have been significantly altered, along with the default size for "solid" changing from 20 to 24, "mini" is effectively the new version of solid @ 20.

**Comment by adamchainz** [comment]
> Nah, I ran the script with `python download_heroicons.py 2.0.0`. I'll look at making the script do the zip remapping. Is it important that the script continues to work for the pre-2.0 layout?

No, we can remove 1.0 support. I think I'll probably just stop supporting the 1.x icons.

> I'd suggest a major version bump for this, as the icons sets have been significantly altered, along with the default size for "solid" changing from 20 to 24, "mini" is effectively the new version of solid @ 20.

Agree.

Could be useful to cover this in the changelgo.

**Pull request merged**